src/core/components/libp2p.js

```diff
@@ -14,9 +15,9 @@ module.exports = function libp2p (self) {
         }
 
         const options = {
-          mdns: config.Discovery.MDNS.Enabled,
-          webRTCStar: config.Discovery.webRTCStar.Enabled,
-          bootstrap: config.Bootstrap
+          mdns: get(config, 'Discovery.MDNS.Enabled'),
+          webRTCStar: get(config, 'Discovery.webRTCStar.Enabled'),
+          bootstrap: get(config, 'Bootstrap')
```

Comment:
This is so that undefined chains don't throw. This code explodes if you try to read a config from go-ipfs as it doesn't have a `webRTCStar` field in discovery.